client/components/place/place-list/place-list.component.js

```diff
@@ -107,6 +111,7 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         ctrl.hidePopularTracks = true;
 
         ctrl.checkPopularPlaces = function() {
+          angular.element('.popular-places-wrapper').css('display', 'block');
```

Comment:
Try adding `.ng-hide` class inside HTML and remove this line and `display: none` inside CSS)